modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts

```diff
@@ -109,25 +119,30 @@ export default class I3SNodePagesTiles {
       lodSelection,
       obb: node.obb,
       contentUrl,
+      // @ts-ignore
       textureUrl,
       attributeUrls,
+      // @ts-ignore
       materialDefinition,
+      // @ts-ignore
       textureFormat,
       textureLoaderOptions: this.textureLoaderOptions,
       children,
+      // @ts-ignore
       isDracoGeometry
     });
   }
 
   /**
    * Forms url and type of geometry resource by nodepage's data and `geometryDefinitions` in the tileset
-   * @param {Object} meshGeometryData - data about the node's mesh from the nodepage
-   * @returns {Object} -
+   * @param - data about the node's mesh from the nodepage
+   * @returns -
    *   {string} url - url to the geometry resource
    *   {boolean} isDracoGeometry - whether the geometry resource contain DRACO compressed geometry
    */
-  private getContentUrl(meshGeometryData) {
-    let result = {};
+  private getContentUrl(meshGeometryData: meshGeometry) {
```

Comment:
CapitalCase for types.

```suggestion
  private getContentUrl(meshGeometryData: MeshGeometry) {
```

Reply:
Done